Makefile

```diff
@@ -33,8 +33,12 @@ install-config:
 	@install -m400 pkg/testing-ecdsa.key $(CONFIG_PREFIX)/keys/testing-ecdsa.key
 	@install -m400 pkg/testing-rsa.key $(CONFIG_PREFIX)/keys/testing-rsa.key
 
-$(INSTALL_BIN)/$(NAME): | install-config
-	@GOOS=linux GOARCH=$(ARCH) go build -o $@ ./cmd/$(NAME)/...
+.PHONY: get-gvt
+get-gvt:
+	go get -u github.com/FiloSottile/gvt
```

Comment:
I agree with the effort 100%, but without aggressive fsck settings, git offers no integrity against malicious adversaries.

If the goal is just to pin the version, I think that's not what that make target wants.


Sent from a small keyboard